config/schema.go

```diff
@@ -157,13 +157,15 @@ var schema = `{
         "directory": {"type": "string"},
         "fstype": {"type": "string"},
         "dev": {"type": "string"},
+        "opt": {"$ref": "#/definitions/list_of_strings"},
         "wait": {"type": "boolean"},
         "required": {"type": "boolean"},
         "autoformat": {"$ref": "#/definitions/list_of_strings"},
         "mdadm_scan": {"type": "boolean"},
         "script": {"type": "string"},
         "oem_fstype": {"type": "string"},
         "oem_dev": {"type": "string"}
+        "oem_opt": {"$ref": "#/definitions/list_of_strings"},
```

Comment:
It looks like this file is out of sync with what's in `schema.json`. You can run `go run scripts/inline_schema.go` to correct this.